WordPress/Classes/ViewRelated/Post/Scheduling/CalendarCollectionView.swift

```diff
@@ -75,34 +136,66 @@ extension CalendarDataSource: JTACMonthViewDelegate {
     }
 
     func calendar(_ calendar: JTACMonthView, didSelectDate date: Date, cell: JTACDayCell?, cellState: CellState, indexPath: IndexPath) {
+        if style == .year {
```

Comment:
Open to suggestions on how to improve this beauty.

Reply:
oof, I spent some time on this but didn't come up with anything better, I also looked at some other projects and they're all doing something similarly. 🤷‍♀️